article/app/views/liveBlog.scala.html

```diff
@@ -10,6 +10,21 @@
     @model.currentPage.pagination.flatMap(_.newer).map(_.suffix).map { suffix =>
         <link rel="prev" href="@LinkTo{ /@{model.article.content.metadata.id}@suffix }">
     }
+    <script type="application/ld+json">
+        @Html(views.structuredData.Organisation().toString())
+    </script>
+
+    <script>console.log('The page was @request.getQueryString("page")')</script>
+
+    <script type="application/ld+json">
+        @Html(views.structuredData.LiveBlogPosting(
+            model.article,
+            LiveBlogHelpers.blocksForLiveBlogRequest(model.article, request.getQueryString("page"))
+        ).toString())
```

Comment:
The LiveBlogPosting returns a jsValue rather than html/string, which makes sense in this context because those classes are dumb json holder objects which are supposed to be embeddable into each other (Like the LiveBlogPosting contains a BlogPosting object) rather than HTML generators.

Wrapping it in @html is required to make it not encode the results.